presto-main-base/pom.xml

```diff
@@ -13,7 +13,7 @@
 
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
-        <project.build.targetJdk>8</project.build.targetJdk>
+        <project.build.targetJdk>17</project.build.targetJdk>
```

Comment:
We want to keep all of these projects at bytecode level of JDK 8 for now. See [RFC discussion](https://github.com/prestodb/rfcs/blob/main/RFC-0010-jdk-upgrade.md#workaround) reg this. A short summary -
1. Any Presto on Spark downstream dependency must be on JDK 8
2. Some of the core client facing modules, like `presto-spi`, `presto-client`, `presto-common` will always be kept a bytecode level of JDK 8 to provide broader support for clients